frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart

```diff
@@ -48,7 +48,10 @@ class SidebarNewPageButton extends StatelessWidget {
       height: 60,
       child: TopBorder(
         color: Theme.of(context).dividerColor,
-        child: child,
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16),
```

Comment:
Where did 16 come from?

Reply:
The original was 12 pixels and then an additional 6. I suppose it should be 18? It seems a bit off when zooming into the screenshot above

Reply:
Oh got it. I will make it 18 then.